src/GraphQL/Extensions/ObjectExtensions.cs

```diff
@@ -166,7 +171,8 @@ string GetPropertyName(string key, out FieldType? field)
 
                 if (propertyInfo != null && propertyInfo.CanWrite)
                 {
-                    object? value = GetPropertyValue(item.Value, propertyInfo.PropertyType, field?.ResolvedType);
+                    object? value = GetPropertyValue(item.Value, propertyInfo.PropertyType, field.ResolvedType
+                        ?? throw new InvalidOperationException($"Could not get ResolvedType for field '{field.Name}' of type '{inputGraphType}'."));
```

Comment:
You use two different messages to say the same thing here and in line 133:
```
"Could not find resolved type for field '{fieldName}' of type '{inputGraphType}'."
"Could not get ResolvedType for field '{field.Name}' of type '{inputGraphType}'."
```

Reply:
Fixed